src/Altinn.App.Api/Controllers/OptionsController.cs

```diff
@@ -34,16 +34,16 @@ public OptionsController(IAppOptionsService appOptionsService)
         /// Api that exposes app related options
         /// </summary>
         /// <param name="optionsId">The optionsId</param>
+        /// <param name="queryParams">Query parameters supplied</param>
         /// <param name="language">The language selected by the user.</param>
-        /// <param name="queryParams">Query parameteres supplied</param>
         /// <returns>The options list</returns>
         [HttpGet("{optionsId}")]
         public async Task<IActionResult> Get(
             [FromRoute] string optionsId,
-            [FromQuery] string? language,
-            [FromQuery] Dictionary<string, string> queryParams)
+            [FromQuery] Dictionary<string, string> queryParams,
+            [FromQuery] string? language = null)
         {
-            AppOptions appOptions = await _appOptionsService.GetOptionsAsync(optionsId, language ?? "nb", queryParams);
```

Comment:
Agreeing that changing this from default "nb" to null makes sense, but in doing so we need to make a new version of https://github.com/Altinn/codelists-lib-dotnet. Some external codelists might expect "nb" as default, but that should be an implementational detail if each `IAppOptionsProvider` implementation.
If we actually decide to remove the nb-default we should create an issue in that repo